src/WarpedPsf.cc

```diff
@@ -111,33 +109,34 @@ afw::geom::Box2I computeBBoxFromTransform
  * The input image is assumed zero-padded.
  */
 PTR(afw::detection::Psf::Image) warpAffine(
-    afw::detection::Psf::Image const &im, afw::geom::AffineTransform const &t,
+    afw::detection::Psf::Image const &im, afw::geom::AffineTransform const &srcToDest,
```

Comment:
I think `srcToDest` is far clearer than `t`. I'll add explicit parameter documentation to the function.

Reply:
I try to avoid adding type information to variable names unless it is needed in order to disambiguate, e.g. if the same mathematical transformation is represented by multiple classes in the same function (a Transform and an AffineTransform, say). That is not an issue in this function.